.github/workflows/playwright.yml

```diff
@@ -13,10 +13,6 @@ jobs:
     runs-on: ubuntu-latest
     container:
       image: mcr.microsoft.com/playwright:v1.41.2-jammy
-    # TODO: Refactor action to use updated logic for Login.gov
-    # when a service account to run tests is available for use.
-    # Issue: CRASH-337
-    if: true == false
```

Comment:
It's unclear to me why this comment is being removed- can someone please explain?

Reply:
A service account has been provided. We needed a test account/mailbox so the testing scripts in Github Actions would not be using an individual user account. We added the new account credentials in Github Secrets and the TODO is resolved.

Reply:
Great, thanks for the clarification @JCantu248. 👍